assets/js/silvermine-videojs-quality-selector.min.js

```diff
@@ -1,3 +1,3 @@
-/*! @silvermine/videojs-quality-selector 2019-09-26 v1.2.2-4-gc134430-dirty */
```

Comment:
You'll want to use [this fork](https://github.com/omarroth/videojs-quality-selector), which has a fix for #158 and silvermine/videojs-quality-selector#57 applied. You can grab a pre-built copy [here](https://omar.yt/js/silvermine-videojs-quality-selector.min.js).

Reply:
Updated.